sql/src/test/java/org/opensearch/sql/sql/parser/AstExpressionBuilderTest.java

```diff
@@ -523,6 +523,7 @@ public void relevanceSimple_query_string() {
             + "analyzer='keyword', operator='AND')"));
   }
 
+
```

Comment:
formatting.

Reply:
Fixed in 7add05fe7c1a7fcef259e1e10656f3c09e50bbc8